schemas/test/en/test_dynamic_answers_list_source.json

```diff
@@ -0,0 +1,462 @@
+                                "when": {
+                                    "==": [
+                                        {
+                                            "count": [
+                                                {
+                                                    "source": "list",
+                                                    "identifier": "supermarkets"
+                                                }
+                                            ]
+                                        },
+                                        0
+                                    ]
+                                }
+                            },
+                            "question": {
+                                "guidance": {
+                                    "contents": [
+                                        {
+                                            "description": "Answers are validated against values piped from previous questions, maximum from repeated question for each supermarket, minimum from non-repeated question. Answer label’s placeholders are resolved from these as well."
+                                        }
+                                    ]
+                                },
+                                "dynamic_answers": {
+                                    "values": {
+                                        "source": "list",
+                                        "identifier": "supermarkets"
+                                    },
+                                    "answers": [
+                                        {
+                                            "label": {
+                                                "text": "How much do you spend at {transformed_value}",
+                                                "placeholders": [
+                                                    {
+                                                        "placeholder": "transformed_value",
+                                                        "value": {
+                                                            "source": "answers",
+                                                            "identifier": "supermarket-name"
+                                                        }
+                                                    }
+                                                ]
+                                            },
+                                            "id": "amount-of-shopping",
+                                            "mandatory": false,
+                                            "type": "Currency",
+                                            "currency": "GBP",
+                                            "decimal_places": 2,
+                                            "description": {
+                                                "text": "The answers must be less than {max_value} and greater than {min_value}",
```

Comment:
`less than or equal {max_value} and greater than or equal {min_value}`. Current equals would work. Either change this or the validation.